### ISSUE TITLE
A mysterious warning

### ISSUE THREAD

**Issue by majdzr** [allegroai/trains#67]
Hello,

I have a weird problem with my code, that only appears when connecting trains to the code. I'm training with tf.keras 2.0, using .fit_generator. I receive the following warnings at the **end of each batch** , and I can't really locate its source in the code:

```
2019-12-17 15:59:43,066 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,071 - trains - WARNING - boolean index did not match indexed array along dimension 1; dimension is 2 but corresponding boolean dimension is 32
2019-12-17 15:59:43,075 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,077 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,078 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,080 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,081 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,083 - trains - WARNING - boolean index did not match indexed array along dimension 1; dimension is 2 but corresponding boolean dimension is 32
2019-12-17 15:59:43,086 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,088 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,089 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,090 - trains - WARNING - too many indices for array
2019-12-17 15:59:43,092 - trains - WARNING - too many indices for array
```
Once I disconnect trains, the warnings disappear. I'm not sure if this is related but I also notice that trains only logs the validation loss and metric, while totally ignoring the training ones. 

Any idea what is causing this or where should I dig to find better clues?
Thanks!
Majd

**Comment by JDennisJ** [comment]
Hi @majdzr 

Thanks for catching this bug. 
We also caught this issue, so a fix is scheduled to be released in one of the coming versions.
I'll update here once it will be released :)

**Comment by majdzr** [comment]
You're quite welcome. Thanks to you and your team for the wonderful product.

One question, is this warning also related to the fact that only one loss is shown or should I dig me?

**Comment by JDennisJ** [comment]
Hi @majdzr ,

You don't need to dig more in the code, I already found the issue and soon will release a fix for it.

I will keep you posted once we have an RC with the fix :smiley:

**Comment by bmartinn** [comment]
Hi @majdzr,
Just letting you know this issue was fixed in v0.13, please update if the warnings consist.

`$ pip install trains>=0.13`

**Comment by majdzr** [comment]
I will. Thanks for taking care of it.
